src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -328,6 +330,23 @@ void GeneralPage::initLayout(GeneralPageView &layout)
 
     layout.addCheckbox("Remove spaces between emotes",
                        s.removeSpacesBetweenEmotes);
+    layout.addCheckbox("Show unlisted / unapproved emotes (7TV only)",
```

Comment:
A little rename, matches documentation here: https://github.com/Chatterino/wiki/pull/235

```suggestion
    layout.addCheckbox("Show unlisted 7TV emotes",
```